runelite-client/src/main/java/net/runelite/client/util/Text.java

```diff
@@ -56,6 +57,18 @@ public static String standardize(String str)
 		return removeTags(str).replace('\u00A0', ' ').trim().toLowerCase();
 	}
 
+	/**
+	 * Convert a string into Jagex username format
+	 * Remove all non-ascii characters, replace nbsp with space, replace _- with spaces, and trim
+	 *
+	 * @param str The string to standardize
+	 * @return The given `str` that is in Jagex name format
+	 */
+	public static String toJagexName(String str)
```

Comment:
We already have method that replaces nbsp with space, 1 line above, is this one really necessary?